example/buf.yaml

```diff
@@ -1,4 +1,9 @@
 version: v1
+
 lint:
   use:
+    - DEFAULT
+  allow_comment_ignores: true
```

Comment:
Yeah feels like everyone wants this feature :)

Reply:
That's the goal! And it does work without making an changes to `rules_lint` directly, just adding that flag does it.